Inventory Management System/Product.cs

```diff
@@ -1,12 +1,88 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Text;
+using System.Text.RegularExpressions;
 using System.Threading.Tasks;
+using System.Xml.Linq;
+using ConsoleTables;
 
 namespace Inventory_Management_System
 {
     internal class Product
     {
+        public string Name { get; set; }
+        public double Price { get; set; }
+        public int Quantity { get; set; }
+        public Product()
+        {
+            this.Name = null;
+            this.Price = 0;
+            this.Quantity = 0;
+        }
+        
+        public void Input()
+        {   //Validating the input to ensure its correctness
+            bool IsValid = false;
+            
+           
+            do
+            {
+                string Name;
+                // Inform the user to enter the product's name
+                Console.Write("Enter the product's name: ");
+                Name = Console.ReadLine();
+                if(Regex.IsMatch(Name, @"^[A-Za-z]+$"))
```

Comment:
please move the Regex value to a helper, because it will be used in many places.